src/entities/SecurityToken/Documents.ts

```diff
@@ -55,17 +57,19 @@ export class Documents extends SubModule {
       );
     } catch (err) {
       throw new PolymathError({
-        code: ErrorCode.ProcedureValidationError,
+        code: ErrorCode.FetcherValidationError,
         message: `There is no Security Token with symbol ${symbol}`,
       });
     }
-    return securityTokenInstance.getDocument(documentName);
+    const documentInfo = await securityTokenInstance.getDocument(args);
+
+    return { name: args.name, updatedAt: documentInfo.documentTime, ...documentInfo };
```

Comment:
This can (and should) be written as

```
const { documentTime: updatedAt, ...info } = await securityTokenInstance.getDocument(args);

return { updatedAt, ...args, ...info };
```

Otherwise you're adding the `documentTime` property to the returned object